src/utils/private.ts

```diff
@@ -68,6 +69,13 @@ export interface CommandObj {
   enabled: boolean;
 }
 
+export interface NewContributorGreeting {
+  enabled: boolean;
+  header: string;
+  helpMenu: boolean;
```

Comment:
I don't understand why you define a new interface: `NewContributorGreeting` here. you've already defined the type `NewContributorGreeting`. lets use it.

Reply:
Because yaml config has kebabCase and internal config has camelCase

Reply:
man now i have to change everything back